core/scrollbar.js

```diff
@@ -196,7 +201,9 @@ Blockly.Scrollbar = function(workspace, horizontal, opt_pair, opt_class) {
   this.createDom_(opt_class);
 
   /**
-   * The upper left corner of the scrollbar's svg group.
+   * The upper left corner of the scrollbar's svg group in CSS pixels relative
```

Comment:
Here's an example, and I'll try to figure out how to word the comments better:
If the flyout is on the right side of the workspace, the *origin* will be something other than (0, 0).  Let's say it's at (250, 0).  So the origin depends on where the flyout was placed on the screen.

The *position* is inside the workspace and depends on the size of the flyout.  If the flyout is 100px wide and the scrollbar is scrolled all the way up, the scrollbar's *position* would be (95, 2.5).  The extra (-5, 2.5) comes from the padding around the scrollbar to make it look nice in the flyout.

The same flyout on the left side of the screen would have the origin at (0, 0) but the position would still be (95, 2.5).